### ISSUE TITLE
代码一个小建议:

### ISSUE THREAD

**Issue by JinYu1998** [zhengli97/CTKD#3]
请删除 `models/__init__.py` 里面的第 **14** 行
```python
from .temp_mlp import Temp_MLP1
```
然后如果方便的话,请提供一个`Requirements.txt`

**Comment by zhengli97** [comment]
谢谢建议~~ 这周末结束前我会update一下。到时候再告知你~

**Comment by Horace-Leon** [comment]
请问一下，为什么代码实现KD时，教师模型的输出是经过log_softmax
KD_loss += nn.KLDivLoss(reduction='batchmean')(F.log_softmax(y_s/T, dim=1), F.softmax(y_t/T, dim=1)) * T * T

**Comment by JinYu1998** [comment]
> 

好的~

**Comment by JinYu1998** [comment]
> 谢谢建议~~ 这周末结束前我会update一下。到时候再告知你~

还有一个问题. `GRL` 对 `loss_div`里面的温度求导的时候.  

```python
loss_div = nn.KLDivLoss()(
F.log_softmax(y_s/T,dim=1),
F.softmax(y_t/T, dim=1)
) * T**2
```
是对 `softmax`里面的温度求导呢? 还是对外面的系数(T**2)求导呢?

**Comment by zhengli97** [comment]
代码已经fix。
GRL反向的时候是算的整体KL loss，所以是外面和里面的T都会涉及。

**Comment by zhengli97** [comment]
有更多想法想交流的话，欢迎加我微信zhengli_97

**Comment by JinYu1998** [comment]
> 代码已经fix。 GRL反向的时候是算的整体KL loss，所以是外面和里面的T都会涉及。

好的~ 谢谢~
